.goreleaser.yml

```diff
@@ -9,6 +9,9 @@ before:
 builds:
   - id: operator-controller
     main: ./
+    hooks:
+      pre:
+        - mkdir -p bin
```

Comment:
Is `bin` only a concern for `goreleaser` because of the `make build` task? If so, WDYT about calling `mkdir -p bin` in the `make build` recipe prior to calling `goreleaser`?

Reply:
It's needed for everything that runs `goreleaser`, including the `release` target, so I figured I'd build it into the yaml so it would always be there regardless of how/which `make` was called.

Reply:
> including the release target

Ah, I didn't know this. How does it come into play with `make release`? (I thought that target used `<projectRoot>/dist` exclusively.

Reply:
I stand corrected, you were right on this one; I've updated it to just call `mkdir -p bin` when running the `build` target.